addons/mail/models/mail_mail.py

```diff
@@ -446,7 +446,7 @@ def send(self, auto_commit=False, raise_exception=False):
                     raise_exception=raise_exception,
                     smtp_session=smtp_session)
                 _logger.info(
-                    'Sent batch %s emails via mail server ID #%s',
+                    "Processed batch of %s mail.mail records via mail server ID #%s",
```

Comment:
I wouldn't change the format of messages in stable unless necessary, that's stuff for master

Reply:
So you suggesting:
- no touchy here
- create a separate PR in Master with this change?

I am fine with that

Reply:
Created PR in master
https://github.com/odoo/odoo/pull/207648